openedx/core/lib/request_utils.py

```diff
@@ -134,7 +134,6 @@ def process_exception(self, request, exception):
 #     attribute, whether they are expected or not. Those errors that are processed and match a 'MODULE_AND_CLASS'
 #     (documented elsewhere), will get an ``error_expected`` custom attribute. Unexpected errors would be errors with
 #     ``error_expected IS NULL``. For additional diagnostic information for ignored errors, see the
-#     IGNORED_ERRORS[N]['IS_IGNORED'] annotation.
```

Comment:
This is part of sentence in the documentation, and the entire documentation for this setting needs to be updated.  Here is a possible replacement:
```
# .. setting_name: IGNORED_ERRORS
# .. setting_default: None
# .. setting_description: Used to configure logging and monitoring for ignored errors.
#     This setting is configured as a list of dicts. See setting and toggle annotations for
#     ``IGNORED_ERRORS[N]['XXX']`` for details of each item in the dict.
#     If this setting is a non-empty list, all uncaught errors processed will get a ``checked_error_ignored_from``
#     attribute, whether they are marked as ignored or not.  For ignored errors, the custom attributes
#      ``error_ignored_class`` and ``error_ignored_message`` are added to help diagnose issues with ignored errors, since
#      this data is not otherwise available. For an example of ignoring errors in New Relic, see:
#      https://docs.newrelic.com/docs/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected/#ignore  pylint: disable=line-too-long,useless-suppression
#      To query for ignored errors, you would use ``error_ignored_class IS NOT NULL``.
# .. toggle_warning: At this time, this toggle does not actually configure the error to be ignored. It is meant to match
#     the ignored error configuration found elsewhere. When monitoring, no errors should ever have the attribute
#     ``error_ignored_class``. Only Transactions should have this custom attribute. If found for an error, it means we
#     are stating an error should be ignored when it is not actually configured as such, or the configuration is not
#     working.  We use Django Middleware and a DRF custom error handler to find uncaught errors. Some errors may
#     slip through the cracks, like ValidationError. Any error where ``checked_error_ignored_from IS NULL`` is
#     an error that was not processed.
```

Changes I made:
1. I updated from `checked_error_expected_from` to `checked_error_ignored_from`, although the code update hasn't yet been made. 
2. I removed the section about the `error_expected` custom attribute. I think this code will simply go away, since it was only for when `IGNORED_ERRORS[N]['IS_IGNORED']` was False.
3. Moved the old documentation you deleted below into this documentation.

Note: Some of my lines may be too long, because I was just editing from github.